src/js/components/awardv2/idv/InfoTooltip.jsx

```diff
@@ -51,15 +51,22 @@ export default class InfoTooltip extends React.Component {
 
     closeTooltip() {
         this.setState({
-            showInfoTooltip: false
+            showInfoTooltip: true
```

Comment:
`closeTooltip` should continue to hide the tooltip - this is causing them to stay open indefinitely.

Reply:
Yes I forgot to change this back! Good catch.